src/components/screens/Consul/Debates/NewDebate.js

```diff
@@ -0,0 +1,180 @@
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import { Alert } from 'react-native';
+import { useForm } from 'react-hook-form';
+import { useMutation } from 'react-apollo';
+
+import { Input } from '../../../Consul';
+import { Wrapper } from '../../../Wrapper';
+import { texts } from '../../../../config';
+import { Button } from '../../../Button';
+import { Checkbox } from '../../../Checkbox';
+import { START_DEBATE } from '../../../../queries/Consul';
+import { ConsulClient } from '../../../../ConsulClient';
+import { LoadingSpinner } from '../../../LoadingSpinner';
+import { ScreenName } from '../../../../types';
+import { QUERY_TYPES } from '../../../../queries';
+import { UPDATE_DEBATE } from '../../../../queries/Consul/Debates/updateDebate';
+
+const text = texts.consul.startNew;
+const queryTypes = QUERY_TYPES.CONSUL;
```

Comment:
this makes it more difficult to read for me at later points. we handle constants here, so they should be uppercased by convention. with having it shorter now but camelCased, it confuse me on reading where used.

why not `QUERY_TYPES.CONSUL.START_DEBATE` and `QUERY_TYPES.CONSUL.UPDATE_DEBATE` where needed?

Reply:
Yes you are right. I updated the code. I will upload the new version soon.

Reply:
can you link to the commit in the new branch please where you made this adjustment?

Reply:
647419dc07997fd7e0f0736c4ec8c98b6a93967e

Is that what you mean?

Reply:
looks good